thrift/thrift0.13/src/test/thrift/Test.thrift

```diff
@@ -0,0 +1,13 @@
+namespace java testing.thrift.test
+
+struct TestServiceRequest {
+  1: string message,
+}
+
+struct TestServiceResponse {
+  1: string response
+}
+
+service TestService {
```

Comment:
We may reuse `HelloService` in main.thrift instead of creating a new one.

Reply:
Fixed it! thank you